crates/bevy_gltf/src/loader.rs

```diff
@@ -211,42 +214,94 @@ fn load_node(
 ) -> Result<(), GltfError> {
     let transform = node.transform();
     let mut gltf_error = None;
-    world_builder
-        .spawn((
+
+    // create camera node
+    let world_builder = if let Some(camera) = node.camera() {
+        match camera.projection() {
+            gltf::camera::Projection::Orthographic(orthographic) => {
+                let xmag = orthographic.xmag();
+                let ymag = orthographic.ymag();
+                let mut projection: OrthographicProjection = Default::default();
+                projection.left = -xmag;
+                projection.right = xmag;
+                projection.top = ymag;
+                projection.bottom = -ymag;
+                projection.far = orthographic.zfar();
+                projection.near = orthographic.znear();
+                projection.get_projection_matrix();
+                world_builder.spawn(Camera2dComponents {
+                    camera: Camera {
+                        name: Some(base::camera::CAMERA2D.to_owned()),
+                        projection_matrix: projection.get_projection_matrix(),
+                        ..Default::default()
+                    },
+                    transform: Transform::from_matrix(Mat4::from_cols_array_2d(
```

Comment:
the transform creation is shared across both projections and the `else` statement. can we move this to a variable on line 217?

Reply:
Job done 😁